packages/patternfly-4/react-core/src/components/Accordion/Accordion.js

```diff
@@ -3,17 +3,34 @@ import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Accordion/accordion';
 
-const Accordion = ({ children, className, 'aria-label': ariaLabel, ...props }) => (
-  <dl className={css(styles.accordion, className)} aria-label={ariaLabel} {...props}>
-    {children}
-  </dl>
-);
+export const AccordionContext = React.createContext({ headingLevel: 'h3' });
+
+export const HeadingLevelTypes = {
```

Comment:
Why is this needed? I think we can just pass `headingLevel` through into context

Reply:
Do you mean I should just set `AccordionContext = React.createContext();` ?

Reply:
I think this const is good to have for the consumer (it'll be the same as enums we export elsewhere, when this is ported over to the typescript version) but Joachim is saying that on line 18 we don't need `const HeadingLevel = HeadingLevelTypes[headingLevel];`, we can just pass lowercase-h `headingLevel` into the provider. The HeadingLevelTypes object doesn't really do anything for us internally, it's just for the consumer